### ISSUE TITLE
Warn users when using BatchNorm with small batch size

### ISSUE THREAD

**Issue by jdeschamps** [CAREamics/careamics#488]
## Problem

Currently, BatchNorm layers are present in the models, and users can set whichever batch size they want. We should probably have the configuration throw a warning if batch size is `<32` or `<8` when using batch norm, and point to deactivating the batch norm layers (possible since a recent PR).

Opinions?

**Comment by melisande-c** [comment]
Have we done much experimenting with this?

**Comment by jdeschamps** [comment]
Not with CAREamics personally, it was reported by users that tend to turn it off. It'd be nice to take the time to have a look on various datasets and for various algorithms as well.

**Comment by jdeschamps** [comment]
Closing because the PR was merged before I saw your message here. I am happy to discuss it again and run some tests (if time allows).
